### ISSUE TITLE
Error Installing: SceDownload change_state DC0 failed: 0x80103a19.

### ISSUE THREAD

**Issue by mpenaranda96** [blastrock/pkgj#354]
Hardware: PS Vita
Version of firmware: 3.60
Version of PKGj: 0.53, 0.54, 0.55

## Steps to reproduce the issue

Trying to download any game from PKGJ

Obtained result: SceDownload change_state DC0 failed: 0x80103a19.
Expected result: Download the game

Hi. This happened after downloading a couple DLC for Senran Kagura Estival Versus, ever since then I haven't been able to download anything from PKGJ. I've tried many fixes that I've read online such as re installing Nodrm, updated PKGJ, installed an older version of PKGJ. Nothing seems to work. I can get by on games with Nopaystation but it's much more convenient to download them from PKGJ. Any help would be appreciated

**Comment by ElesonTan84** [comment]
Make sure you have nonpdrm plugin installed.

**Comment by VegetableYummee** [comment]
I was having the same problem, I reinstalled PKGj v.0.55 also installed the fixed version and nothing happened, same error. After trying many times and reading a lot I realized I had my Vita on airplane mode.........as soon as I activated the Wi-Fi again, I was able to download the games.......you can laugh now.

**Comment by arkonshin** [comment]
any way how to fix this problem??

**Comment by plumberg** [comment]
> any way how to fix this problem??

Yup, solution to this issue was pretty dumb -- check available space of your memory card. There should be enough for your new game.

**Comment by spoergsmaal** [comment]
> I was having the same problem, I reinstalled PKGj v.0.55 also installed the fixed version and nothing happened, same error. After trying many times and reading a lot I realized I had my Vita on airplane mode.........as soon as I activated the Wi-Fi again, I was able to download the games.......you can laugh now.

yep, this happened to me too

**Comment by NeriNewton** [comment]
Hey there, I have 8 Gb of free memory space, the game I want is like 3.3 Gb, I have Nonpdrm, the airplane mode is off, the Wi-Fi is on, everything is correct, install and delete older versions, I try everything guys and I still have the Error Installing: SceDownload change_state DC0 failed: 0x80103a19. Any other suggestions? Thanks!

**Comment by iwantcustom** [comment]
Hello there!
I had the same problem. I surfed the net and found nothing about it and I was freaking out. HOWEVER I tried changing the DNS adress to auto(I set it to smth while following a guide) and it worked! Hope this helps

**Comment by gibo03** [comment]
yup its just a network issue. Either you are in airplane mode or your wifi does not provide download, with that I mean you have signal but you cant access internet

**Comment by NeriNewton** [comment]
Hello! I had had the same problem for a long time and was already desperate, I tried one last thing, according to the settings, the SD memory was not detected correctly, I removed it, rebooted and everything worked perfect, The old trick of removing the memory, inserting it, and rebooting.  I wish you luck and that you can fix it soon.

**Comment by DougNg** [comment]
Mine was dumb.  I forgot that I turned on Flight Mode to save battery.
I guess the solution is either: Internet issue or storage

**Comment by nabetse00** [comment]
For me this worked:
- boot the vita
- unplug sd2vita
- reconnect sd2vita

You should see the orange led blinking (don't put the plastic cap on), wait until it stops.
Retry installation it should work if internet connection is available.

I think it is a edge bug when your data is the same in sd2vita card as memory card and no internet is available: it tries to read it from sony memory until sd2vita is reinstalled (not on reboot); I have no proof this theory is correct, except that unplug replug worked.

Hope it helps someone !

**Comment by Masamune3210** [comment]
plllleeease dont tell people to remove a currently mounted filesystem randomly as a fix, especially if you arent 100% sure that it will do anything.   At best you get crashes due to apps files just up and disappearing out from under them, at worst you get a corrupted card as a present

**Comment by nabetse00** [comment]
@Masamune3210 you are right. Updated my comment with caution warnings.
Nevertheless i didn't told anyone to do anything. I was just sharing the process that worked for my particular situation.

**Comment by cathD09** [comment]
I gotta be mad with this problem. I have enough space, not on Flight Mode, have Nonpdrm, my wifi is on; I tried taking off my SD then put it back too. But nothing changes! And when I copy games through USB, it can't be installed cause the "error 0x80af5500" . 

If my vita tried to against me, it could be successful. Can everyone give me more advices, please?

**Comment by Masamune3210** [comment]
If you are getting a error even when manually copying files to the vita and trying to install them, then your issue likely lies elseware, possibly a corrupted or improperly set up sd card

**Comment by cathD09** [comment]
> If you are getting a error even when manually copying files to the vita and trying to install them, then your issue likely lies elseware, possibly a corrupted or improperly set up sd card

Uhm... what can I do with that?

**Comment by Masamune3210** [comment]
Find out why writing to the storage medium is failing......

**Comment by cuevavirus** [comment]
There are various reasons this error could occur even with the Store app. Not likely to be a problem with pkgj.

**Comment by RafaliyoP** [comment]
> ajajajajjajjajajajjajjajjajaja

**Comment by indianDEADman** [comment]
People !!! OFF ENERGY SAVE MODE WIFI ON YOURS PS VITA... ITS THIS MODE STARTING ERROR

**Comment by KaylaD23** [comment]
> I was having the same problem, I reinstalled PKGj v.0.55 also installed the fixed version and nothing happened, same error. After trying many times and reading a lot I realized I had my Vita on airplane mode.........as soon as I activated the Wi-Fi again, I was able to download the games.......you can laugh now.

its a vita bug i think i didnt have airplane mode on but i switched it on then switched back to wifi and it started working very strange

**Comment by MDSamaX** [comment]
Pour ma part j'ai simplement redémarré ma PSVITA, ensuite j'ai réessayé de télécharger un PSVITA et PSP, cela à fonctionné

**Comment by samile2** [comment]
I have delete then re-created the folder 
ux0:bgdl 
With empty folders "t" and "w" and a empty file called "temp.dat"
The file temp.dat seems corrupted for me.

Worked for me.

**Comment by Kawet00** [comment]
how do you re create the folder with goods files?

**Comment by stanyertom** [comment]
> I have delete then re-created the folder ux0:bgdl With empty folders "t" and "w" and a empty file called "temp.dat" The file temp.dat seems corrupted for me.
> 
> Worked for me.

You sir are a legend…this worked for me. Thanks

**Comment by dmytrofrolov** [comment]
For the record, for me it was the internet connection.
Custom DNS, to be precise, was set in router but Vita couldn't work with it apparently.
Internet test in Vita settings was showing no internet with wifi connected.
**I've set custom DNS at Vita settings to 8.8.8.8** and issue resolved by itself.
Checked PKGj 0.55 and 0.57.
Also I did try "ux0:bgdl" fix and deleting other games to free up some space at SD2Vita but obviously it didn't help.
